feature-probe-ui/src/constants/dictionary_keys.ts

```diff
@@ -1,5 +1,5 @@
 export const LAST_SEEN = 'user_last_seen';
 export const DEMO_TIP_SHOW = 'is_demo_tips_show';
-export const USER_GUIDE_LAYOUT = 'user_guide_layout';
+export const USER_GUIDE_LAYOUT = 'user_guide_layout_version_1';
```

Comment:
Why adding a new key here?  can not use `user_guide_layout` directly?

Reply:
Because we should start in first，and this is the easiest way